web/src/engine/websites/ComicEarthStar.ts

```diff
@@ -0,0 +1,121 @@
+import { Tags } from '../Tags';
+import icon from './ComicEarthStar.webp';
+import { DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as CoreView from './decorators/CoreView';
+import * as Common from './decorators/Common';
+import { FetchGraphQL, FetchRequest } from '../FetchProvider';
+
+const queryOneshots = `
+            query Earthstar_Oneshot {
+                seriesOneshot: serialGroup(groupName: "連載・読切：読切作品") {
+                    seriesSlice {
+                        seriesList {
+                            ...Earthstar_SeriesListItem_Series
+                        }
+                    }
+                }
+            }
+            fragment Earthstar_SeriesListItem_Series on Series {
```

Comment:
Fragment is only used once, consider to embed the content directly and drop the fragment